README.md

```diff
@@ -29,7 +29,7 @@ docker run --rm -it -v $PWD:/app -w /app asyncapi/generator [COMMAND HERE]
   Options:
 
     -V, --version                  output the version number
-    -w, --watch                    watches the the template directory and the async api document for change, and re-generate the files when they occour.
+    -w, --watch                    watches the template directory and the async api document for change, and re-generate the files when they occour.
```

Comment:
Now that you're here, can you please fix "async api"? It should be "AsyncAPI" because it's a brand name.